src/mlpack/methods/reinforcement_learning/q_learning.hpp

```diff
@@ -111,6 +111,11 @@ class QLearning
   //! Get the state of the agent.
   const StateType& State() const { return state; }
 
+  //! Modify the action of the agent.
+  ActionType& Action() { return action; }
```

Comment:
Does the setter function do anything here? Correct me if I'm wrong, but i think the action is chosen by the policy anyway, so a user setting an action wouldn't do anything?

Reply:
Hmm, u r right. I just added it thinking maybe we might use it in the future. But lets remove that for now.. Thanks :)

Reply:
BTW, I always get confused. Is it that all PRs, no matter what changes they make, should be added to HISTORY.md? Is there a threshold as such?

Reply:
The threshold is probably different for different people, but imo its when we add new user facing things, it should be in the changelog (or big internal changes too)

Reply:
Oh, I see